gradle.properties

```diff
@@ -2,25 +2,40 @@ rootProject.name=scheduler
 profile=dev
 
 # Build properties
-node_version=10.14.1
-npm_version=6.4.1
-yarn_version=1.12.3
+node_version=12.13.0
+npm_version=6.13.0
+yarn_version=1.19.0
 
 # Dependency versions
-jhipster_dependencies_version=2.0.29
+jhipster_dependencies_version=3.0.7
 # The spring-boot version should match the one managed by
 # https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/${jhipster_dependencies_version}
-spring_boot_version=2.0.7.RELEASE
+spring_boot_version=2.1.10.RELEASE
 # The hibernate version should match the one managed by
 # https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
-hibernate_version=5.2.17.Final
-mapstruct_version=1.2.0.Final
+hibernate_version=5.3.13.Final
+mapstruct_version=1.3.1.Final
+archunit_junit5_version=0.12.0
+log4j2_mock_version=0.0.1
 
 liquibase_hibernate5_version=3.6
 liquibaseTaskPrefix=liquibase
 
+jaxb_runtime_version=2.3.2
+
+# gradle plugin version
+jib_plugin_version=1.7.0
+git_properties_plugin_version=2.2.0
+apt_plugin_version=0.21
+liquibase_plugin_version=2.0.1
+sonarqube_plugin_version=2.8
+spring_no_http_plugin_version=0.0.3.RELEASE
+checkstyle_version=8.24
+
 lombok_version=1.18.2
-xm_commons_version=2.0.21
+xm_commons_version=2.1.0
+spotbugs_version=3.0.0
```

Comment:
move this line to the " gradle plugin version" section

Reply:
done